scripts/azdo/conf/auto.conf

```diff
@@ -0,0 +1,14 @@
+include conf/ni-org.conf
```

Comment:
I'm not convinced that this needs to be a separate conf file from ni-org.conf. This too will not be used by anyone outside NI, right? 
I'm also concerned that for a developer build outside azdo, which is a common-enough case, the configuration in these two files will not be included in the build by default. What are your thoughts about that?

Reply:
I broke out configuration into the auto.conf which enabled unnecessary build artifacts (buildhistory, cve logs, et c.). My thinking was that NI devs can use the auto.conf, but not waste time making those extraneous artifacts, unless they want to explicitly add them to their local.conf.

> This too will not be used by anyone outside NI, right?

Correct.

>I'm also concerned that for a developer build outside azdo, which is a common-enough case, the configuration in these two files will not be included in the build by default. What are your thoughts about that?

I suppose we can add a switch to [`nilrt-build-init.env`](https://github.com/ni/nilrt/blob/nilrt/master/dunfell/nilrt-build-init.env) which adds the ni-org.conf to your build environment automatically. Would that suffice?

Reply:
> I suppose we can add a switch to nilrt-build-init.env which adds the ni-org.conf to your build environment automatically. Would that suffice?

I guess there aren't better alternatives. Can you add a note to the README recommending how to invoke the script for NI developers? And announce this change to the team.